### PR TITLE
Docs: Update REST catalog config for Trino

### PR DESCRIPTION
**Pull request by ajantha-bhat**

### CODE REVIEW

File: site/docs/iceberg/trino.md [hunk 1]
```diff
@@ -28,4 +28,13 @@ s3.aws-secret-key=<aws_secret_key>
        a. To create branches and tags, use the Nessie CLI available [here](https://projectnessie.org/nessie-latest/cli/).
        b. To switch references (branch or tag), update the catalog property `iceberg.nessie-catalog.ref`.
 
+Nessie can also be accessed via REST catalog path.
+Sample Nessie-REST configuration for Iceberg connector in `etc/catalog/iceberg.properties`
+```
+connector.name=iceberg
+iceberg.catalog.type=rest
+iceberg.rest-catalog.uri=http://.../iceberg
```
Comment (dimas-b) [inline site/docs/iceberg/trino.md:36]: nit: I'd use the example URI of `http://localhost:19120/iceberg` - similar to the example above... `...` might be confusing to some users, I guess.

File: site/docs/iceberg/trino.md [hunk 2]
```diff
@@ -28,4 +28,13 @@ s3.aws-secret-key=<aws_secret_key>
        a. To create branches and tags, use the Nessie CLI available [here](https://projectnessie.org/nessie-latest/cli/).
        b. To switch references (branch or tag), update the catalog property `iceberg.nessie-catalog.ref`.
 
+Nessie can also be accessed via REST catalog path.
+Sample Nessie-REST configuration for Iceberg connector in `etc/catalog/iceberg.properties`
```
Comment (dimas-b) [inline site/docs/iceberg/trino.md:32]: nit: Maybe put `:` at the end? (also on the similar line above)

File: site/docs/iceberg/trino.md [hunk 3]
```diff
@@ -28,4 +28,13 @@ s3.aws-secret-key=<aws_secret_key>
...
+```
```
Comment (adutra) [inline site/docs/iceberg/trino.md:33]:
```suggestion
```properties
```

Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md]: nice. TIL about this syntax :D

File: site/docs/iceberg/trino.md (hunk 2 above)
Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md]: done

File: site/docs/iceberg/trino.md (hunk 1 above)
Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md]: done

**Review by dimas-b — APPROVED** [review]

**Pull request merged**